src/densities.jl

```diff
@@ -44,7 +47,7 @@ grid `basis`, where the individual k-Points are occupied according to `occupatio
 `ψ` should be one coefficient matrix per k-Point. If the `Model` underlying the basis
 is not collinear the spin density is `nothing`.
 """
-@timing function compute_density(basis::PlaneWaveBasis, ψ, occupation)
+@views @timing function compute_density(basis::PlaneWaveBasis, ψ, occupation)
```

Comment:
Actually I wondered if we could put a `@views` in front of *the whole of DFTK*: can we do that? We very very rarely (never?) use non-views slices, and when we do it's because we forgot to put a `@views`. The fact that julia's slices copy by default is a design mistake that was done because views used to be slow, and I hope they fix it for 2.0. So I'd rather put a `@views` in front of every function that uses slices and forget about it.

Reply:
I'm not sure about that. I had cases (could be in older Julia versions) where @views caused really subtle issues - both coding-wise and performance-wise, so I tend to use `@views` rather locally.
So definitely I'm against using `@views` for the full DFTK (can probably be done by putting it in front of the module definition) as that would be rather unexpected I'd say.